fedimint-bin-tests/src/main.rs

```diff
@@ -152,12 +153,11 @@ impl Lightningd {
     }
 
     pub async fn cmd(&self) -> Command {
+        let cln_dir = &self.inner.cln_dir;
         let mut cmd = cmd!(
             "lightning-cli",
-            "--network",
-            "regtest",
-            "--lightning-dir",
-            self.inner.cln_dir.clone(),
+            "--network=regtest",
+            "--lightning-dir={cln_dir}",
```

Comment:
Part of the idea of moving these tests to rust is so that we could use Rust clients for commands like this. Then everything is typed. IMO we should only call the shell when we start daemons.

Reply:
Yes, that is the goal. But this was just faster to write. 

I have converted all bitcoin-cli commands to using rpc now.